test/test_create.py

```diff
@@ -0,0 +1,77 @@
+
+from snowflake.sqlalchemy import (
+    AzureContainer,
+    CreateFileFormat,
+    CreateStage,
+    CSVFormatter,
+    ExternalStage,
+    PARQUETFormatter,
+)
+
+
+def test_create_stage(sql_compiler):
+    """
+    This test compiles the SQL to create a named stage, by defining the stage naming
+    information (namespace and name) and the physical storage information (here: an
+    Azure container), and combining them in a CreateStage object
+    NB: The test only validates that the correct SQL is generated. It does not
+    execute the SQL (yet) against an actual Snowflake instance.
+    """
+    # define the stage name
+    stage = ExternalStage(
+        name="AZURE_STAGE",
+        namespace="ML_POC.PUBLIC",
+    )
+    # define the storage container
+    container = AzureContainer(
```

Comment:
Should probably be done, and CREATE STAGE support for GCP could probably be added as well. But since we are only using Azure at the moment, I started with the tests for Azure to cover our use cases.